main/factor.R

```diff
@@ -0,0 +1,45 @@
+# redirect output to a file
+sink("factor.out")
+
+library(readr)
+library(nFactors)
+library(DataCombine)
```

Comment:
Hmm, I don't have access to this library either. Hmm...

Reply:
Adding it to `r_packages.csv`

Reply:
It's in `r_pacakges.csv`. maybe when you run it as a bazel binary you'll be able to use the library?

Reply:
Yes, that's my hope.